src/Api.php

```diff
@@ -244,8 +248,10 @@ private function rawCall(
     /**
      * Wrap call to Ovh APIs for GET requests
      *
-     * @param $path path ask inside api
-     * @param $content content to send inside body of request
+     * @param string $path path ask inside api
+     * @param array\\stdClass|null $content content to send inside body of request
```

Comment:
arf, I just spotted a typo here. The first backslash should have been ``|``. I will send an update